3.6/administration-cluster.md

```diff
@@ -99,8 +99,9 @@ It is also possible to specify multiple `shardKeys`.
 
 Note however that if you change the shard keys from their default `["_key"]`, then finding
 a document in the collection by its primary key involves a request to
-every single shard. However this can be mitigate: All CRUD APIs and AQL
-support taking the shard keys as a lookup hint.
+every single shard. However this can be mitigated: All CRUD APIs and AQL
+support taking the shard keys as a lookup hint. Just send them as part
+of the update / replace or removal operation.
```

Comment:
I assume the respective attributes need to be present in the request for the CRUD APIs?
In case of AQL, does it work with all syntax variants?
- `UPDATE "docKey" WITH { myShardAttribute: "foo", ... }`
- `UPDATE { _key: "docKey", myShardAttribute: "foo", ...}`
- `UPDATE { _key: "docKey", myShardAttribute: "foo" } WITH { ... }` ???
- `UPSERT { _key: "docKey", myShardAttribute: "foo" }` ... ?
- `UPSERT { _key: "docKey" } INSERT { myShardAttribute: "foo", ... } UPDATE { myShardAttribute: "foo", ... }` ?

Reply:
Yes feel free to  add it to the docs yourself. This always used to work reliably except for remove operations, new in 3.6 is only that it now does work everywhere